celery/canvas.py

```diff
@@ -1055,9 +1044,7 @@ def from_dict(cls, d, app=None):
         d["kwargs"]["tasks"] = rebuilt_tasks = type(orig_tasks)((
             maybe_signature(task, app=app) for task in orig_tasks
         ))
-        return _upgrade(
-            d, group(rebuilt_tasks, app=app, **d['options']),
-        )
+        return group(rebuilt_tasks, app=app, **d['options'])
```

Comment:
can we add some comment here explaining the change?

Reply:
`_upgrade` used to just set the chord size which has been refactored out since it is set on the backend now instead.
![image](https://user-images.githubusercontent.com/1480930/108213570-2e414b80-70fd-11eb-955e-bf76320a3d99.png)

Reply:
btw do you have the bandwidth to check the pypy failures? maybe in a different PR. also as we don't have CI for Integration tests now, did all the integration tests pass locally?

Reply:
yep, I can take a look at the pypy failures in a separate PR.  Integration tests are all passing locally

Reply:
Here's a PR I started to look into the PyPy failures: https://github.com/celery/celery/pull/6635.  Need some feedback on it when you get some time @auvipy.